index.js

```diff
@@ -1,17 +1,61 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [],
+    this.length = 0
+  }
 
-  add(item) {}
+  add(item) {
+      this.items.push(item);
+      this.items.sort((a, b) => a - b);
+      this.length ++;
+    
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (typeof this.items[pos] !== 'number'){
+      throw new Error('OutOfBounds');
+    } else {
+      return this.items[pos];
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0){
+      throw new Error('EmptySortedList');
+    } else {
+      return this.items[this.length - 1];
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.length === 0){
+      throw new Error('EmptySortedList');
+    } else {
+      return this.items[0]
+    }
+  }
 
-  sum() {}
+  sum() {
+    if (this.length === 0){
+      return 0;
+    } else {
+      let sum = this.items.reduce((a, b) => {
+        return a + b;
+      })
+      return sum;
```

Comment:
you can reduce an empty array. the result will only be the starting value you pass onto it.

you didnt need to do this if statement and assingnint it to a variable.

```js
return this.items.reduce((acc, val) => {
    acc + val
}, 0)
```